bin/oils_for_unix.py

```diff
@@ -133,10 +133,11 @@ def AppBundleMain(argv):
 
     environ = pyos.Environ()
 
-    if applet in ('ysh', 'oil'):
+    if applet.startswith('ysh') or applet.startswith('oil'):
```

Comment:
let's keep this backward compat check  as `applet == 'oil'`

Because `oil` is the old name of the project, and `oils` starts with `oil`.   I don't want to  people to get confused

Reply:
Thanks! I changed to `applet == 'oil'` in commit 7c90a083574b3f6037be720d5c039751d866ef54.